src/actions/allActivitiesActions.js

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+
+import {
+  FETCH_ALL_ACTIVITIES_FAILURE,
+  FETCH_ALL_ACTIVITIES_SUCCESS,
+  FETCH_ALL_ACTIVITIES_REQUEST,
+} from '../types';
+import config from '../../config';
+
+/**
+ * @function fetchAllPendingRequests
+ * @return {Object} {{type: FETCH_ALL_ACTIVITIES_REQUEST}}
+ */
+export const fetchAllActivitiesRequests = () => (
+  {
+    type: FETCH_ALL_ACTIVITIES_REQUEST,
+  }
+);
+
+/**
+ * @function fetchAllSuccess
+ * @param info - array of fetch all activities
```

Comment:
Refactor the param `info` to match the one you are using in `fetchAllActivitiesSuccess `.